src/tck/Reactive.Streams.TCK.Tests/PublisherVerificationTest.cs

```diff
@@ -566,8 +607,11 @@ public void Subscribe(ISubscriber<int> subscriber)
                     {
                         for (var i = 0L; i <= n; i++)
```

Comment:
@viktorklang yes, the jvm version injections a executor service which is shut downed in the finally block